app/controllers/proposals_controller.rb

```diff
@@ -57,6 +62,11 @@ def destroy
   end
 
   def create
+    if @event.closed? && @event.closes_at < 1.hour.ago
```

Comment:
Can you verify that if the CFP closed 55 minutes ago that it will let the submission through?  That would be an excellent test to write since you'll also be documenting the hour grace period.